ui/package.json

```diff
@@ -61,102 +60,101 @@
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-maybe-in-element": "^0.1.3",
+    "ember-maybe-in-element": "^0.4.0",
     "ember-power-select-with-create": "cibernox/ember-power-select-with-create#6203918f247c1c5d692db4f9185ab8321d2125e1",
     "ember-qunit": "^4.4.1",
-    "ember-radio-button": "^1.1.1",
+    "ember-radio-button": "^2.0.1",
     "ember-resolver": "^5.0.1",
     "ember-responsive": "^3.0.0-beta.3",
     "ember-router-helpers": "^0.2.0",
-    "ember-sinon": "^1.0.1",
-    "ember-source": "~3.4.0",
-    "ember-svg-jar": "^1.2.2",
+    "ember-sinon": "^4.0.0",
+    "ember-source": "~3.8.0",
+    "ember-svg-jar": "^2.1.0",
     "ember-test-selectors": "^2.1.0",
     "ember-truth-helpers": "^2.1.0",
-    "escape-string-regexp": "^1.0.5",
-    "eslint": "^5.16.0",
-    "eslint-config-prettier": "^3.1.0",
-    "eslint-plugin-ember": "^6.3.0",
-    "eslint-plugin-prettier": "^3.0.0",
+    "escape-string-regexp": "^2.0.0",
+    "eslint": "^6.1.0",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-ember": "^6.7.0",
+    "eslint-plugin-prettier": "^3.1.0",
     "flat": "^4.1.0",
-    "ivy-codemirror": "2.1.0",
-    "jsonlint": "1.6.0",
+    "ivy-codemirror": "IvyApp/ivy-codemirror#fb09333c5144da47e14a9e6260f80577d5408374",
+    "jsonlint": "^1.6.3",
     "loader.js": "^4.7.0",
     "node-sass": "^4.10.0",
     "normalize.css": "4.1.1",
     "prettier": "^1.14.3",
-    "prettier-eslint-cli": "^4.7.1",
+    "prettier-eslint-cli": "^5.0.0",
     "qunit-dom": "^0.7.1",
     "route-recognizer": "^0.3.4",
     "sass-svg-uri": "^1.0.0",
     "shell-quote": "^1.6.1",
     "string.prototype.endswith": "^0.2.0",
     "string.prototype.startswith": "^0.2.0",
     "swagger-ui-dist": "^3.22.3",
-    "text-encoder-lite": "1.0.0",
-    "walk-sync": "^0.3.3",
+    "text-encoder-lite": "2.0.0",
+    "walk-sync": "^2.0.2",
     "xstate": "^3.3.3",
-    "yargs-parser": "^13.0.0"
+    "yargs-parser": "^13.1.1"
   },
   "optionalDependencies": {
-    "@babel/core": "^7.3.4",
-    "@storybook/addon-knobs": "^5.0.5",
-    "@storybook/addon-links": "^5.0.5",
-    "@storybook/addon-notes": "^5.0.5",
-    "@storybook/addon-viewport": "^5.0.5",
-    "@storybook/addons": "^5.0.5",
-    "@storybook/ember": "^5.0.5",
-    "@storybook/ember-cli-storybook": "meirish/ember-cli-storybook#6bd58326d8c21e986d390b541ae5e49089d61b93",
-    "babel-loader": "^8.0.5",
-    "jsdoc-to-markdown": "^4.0.1",
-    "lint-staged": "^8.0.4"
+    "@babel/core": "^7.5.5",
+    "@storybook/addon-knobs": "^5.1.10",
+    "@storybook/addon-links": "^5.1.10",
+    "@storybook/addon-notes": "^5.1.10",
+    "@storybook/addon-viewport": "^5.1.10",
+    "@storybook/addons": "^5.1.10",
+    "@storybook/ember": "^5.1.10",
+    "babel-loader": "^8.0.6",
+    "jsdoc-to-markdown": "^5.0.0",
+    "lint-staged": "^9.2.1"
   },
   "resolutions": {
     "handlebars": "^4.1.2"
   },
   "engines": {
-    "node": " >= 10.*"
+    "node": " >= 10.* <11",
```

Comment:
i was using node `10.12.0` and ran `yarn start` and got an error: 
```
The engine "node" is incompatible with this module. Expected version "^8.10.0 || ^10.13.0 || >=11.10.1". Got "10.12.0"
```
i'm not sure why this failed since 10.12.0 is `>=` 10, but maybe we should update this logic and the `.nvmrc`?

Reply:
Sorry, thought I had replied to this, but must have forgotten to submit. I'm not sure what that error is coming from, but it's not what we've specified in vault...